content/docs/connect/connect-from-any-app.md

```diff
@@ -59,6 +59,8 @@ psql postgres://casey:<password>@ep-polished-water-579720.us-east-2.aws.neon.tec
 
 <Admonition type="note">
 For information about connecting with `psql`, see [Connect with psql](../query-with-psql-editor). Please be aware that some older client libraries and drivers, including older `psql` executables, are built without SNI support and require a workaround. For more information, see [Connecting from older clients](../connectivity-issues).
+
+Some tools, such as DBeaver, DataGrip, and CLion, do not support inclusion of a user name and password in a database connection string. When you find that a full connection string is not accepted, try entering the user name and password values in the appropriate fields in the tool's UI when configuring a connection to Neon.
```

Comment:
Maybe add some specifics? Tools affected use the pgJDBC driver, which is the official PostgreSQL driver (library) for connecting to PostgreSQL from Java/Kotlin/other JVM languages. Suggestion: "Some JVM-based tools, such as...", although "Java" would be more recognizable than "JVM", but less precise.